other/materials_designer/interface_creation_new.ipynb

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
<p>Remove this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>